.travis.yml

```diff
@@ -1,40 +1,51 @@
-language: php
+version: ~> 1.0
 
-dist: xenial
-
-services:
-  - mysql
-  - postgresql
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/self.yml
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=4.x-dev
-
-matrix:
+    - COMPOSER_ROOT_VERSION="4.7.x-dev"
+    - REQUIRE_EXTRA="silverstripe-themes/simple:~3.2.0 silverstripe/admin:1.7.x-dev silverstripe/versioned:1.7.x-dev mikey179/vfsstream:^1.6"
+    - PHPUNIT_SUITE="recipe-core"
+
+# Using a manual matrix to exlude PHPUNIT_COVERAGE_TEST which was having an unknown error
+# https://travis-ci.com/github/silverstripe/recipe-core/jobs/457587137
+jobs:
+  fast_finish: true
   include:
     - php: 7.1
-      env: DB=MYSQL COMPOSER_ARG=--prefer-lowest
+      env:
+        - DB=MYSQL
+        - PHPUNIT_TEST=1
     - php: 7.2
-      env: DB=PGSQL
+      env:
+        - DB=PGSQL
+        - PHPUNIT_TEST=1
+        - COW_TEST=1
     - php: 7.3
-      env: DB=MYSQL
+      env:
+        - DB=MYSQL
+        - PDO=1
+        - PHPUNIT_TEST=1
+        - PHPCS_TEST=1
     - php: 7.4
-      env: DB=MYSQL
+      env:
+        - DB=MYSQL
+        #- PHPUNIT_COVERAGE_TEST=1
+        - PHPUNIT_TEST=1
+    - php: nightly
+      env:
+        - DB=MYSQL
+        - PHPUNIT_TEST=1
+        - COMPOSER_INSTALL_ARG=--ignore-platform-reqs
+  allow_failures:
     - php: nightly
```

Comment:
We don't want to allow failures against PHP 8 in core.

Reply:
Have removed this bit